test/metabase/query_processor_test/date_bucketing_test.clj

```diff
@@ -724,6 +724,25 @@
     (is (= [[22 46] [23 47] [24 40] [25 60] [26 7]]
            (sad-toucan-incidents-with-bucketing :week-of-year :utc)))))
 
+(deftest week-of-year-and-week-count-should-be-consistent-test
+  (testing "consistent break out between weeks and week-of-year #4910"
+    (mt/test-drivers (mt/normal-drivers)
+      ;; 2019-01-01 is Tuesday, so set start-of-week to tuesday so
+      ;; breakout by week-of-year will have first row is the 1st week of year
+      (mt/with-start-of-week :tuesday
+       (mt/dataset sample-dataset
+         (letfn [(test-break-out [unit] (->> (mt/mbql-query orders
+                                                {:filter      [:between $created_at "2019-01-01" "2019-12-31"]
+                                                 :breakout    [:field $created_at {:temporal-unit unit}]
+                                                 :aggregation [[:count]]})
+                                           mt/process-query
+                                           (mt/formatted-rows [str int])
+                                           (map second)))]
+          (is (= (test-break-out :week)
+                 (test-break-out :week-of-year)))
```

Comment:
I was very confused at why these were supposed to be equal but then I realized you're only looking at the counts and not the actual values